packages/skygear-web/src/pkce.ts

```diff
@@ -0,0 +1,51 @@
+import { _encodeUTF8, _encodeBase64URLFromUint8Array } from "@skygear/core";
+
+function byteToHex(byte: number): string {
+  return ("0" + byte.toString(16)).substr(-2);
+}
+
+function numberArrayToUint8Array(arr: number[]): Uint8Array {
```

Comment:
We can use `Uint8Array.from(arr)` instead.

Reply:
IE does not have that static function